cgi-bin/LJ/User.pm

```diff
@@ -4949,6 +4958,12 @@ sub update_email_alias {
               undef, $u->site_email_alias, $u->email_raw );
 
     return 0 if $dbh->err;
+
+    $dbh->do( "REPLACE INTO email_aliases (alias, rcpt) VALUES (?,?)",
```

Comment:
If I understand you right this isn't a new problem introduced by this change, because it was true of abc_123 even before. Perhaps this should be another bug?